readme.md

```diff
@@ -50,7 +50,7 @@ import { Context, APIGatewayProxyResult, APIGatewayEvent } from 'aws-lambda'
 export const handler = async (event: APIGatewayEvent, context: Context): Promise<APIGatewayProxyResult> => {
   console.log(`Event:`, event)
   console.log(`Context:`, context)
-  return { message: 'hello world' }
+  return { statusCode: 200, body: JSON.stringify({ message: 'Hello world!' }) }
```

Comment:
Returning a raw object as in this example will be interpreted by API Gateway `HTTP` APIs as valid JSON / 200; is there a reason for this proposed change?

Reply:
Cheers Ryan. The reason is just to get the generated function to typecheck.
The handler signature as defined:
`const handler = async (event: APIGatewayEvent, context: Context): Promise<APIGatewayProxyResult> `
won't typecheck when returning the raw object. Theres a new assertion in one of the tests in the PR that caught this.